addons/content-index/src/main/java/org/commonjava/indy/content/index/ContentIndexObserver.java

```diff
@@ -272,8 +272,8 @@ private void propagatePathlessStoreEvent( Iterable<ArtifactStore> stores )
                 final StoreKey key = store.getKey();
 
                 Set<String> paths = new HashSet<>();
-                indexManager.removeAllIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPath());} );
-                indexManager.removeAllOriginIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPath());} );
+                indexManager.removeAllIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPathHash());} );
+                indexManager.removeAllOriginIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPathHash());} );
 
```

Comment:
This is triggered when a repo is disabled, so it is not related to a path. It should process all indexed paths from the disabled repo.